src/collapsible.tab.js

```diff
@@ -68,14 +68,14 @@
 				e.preventDefault();
 				e.stopPropagation();
 
-				if( self.$tabHeader.is( '.' + activeTabClass ) ) {
-					deactivateTab( self.$tabHeader );
-				} else {
-					deactivateTab( $tabContainer.find( '.' + activeTabClass ) );
+				if( !self.$tabHeader.is( '.' + activeTabClass ) ) {
+					var nonActives = $tabContainer.find( '.' + activeTabClass ).not( self.$tabHeader );
```

Comment:
Two notes here:

1. This line confused me, the selector seems to be for the _active_ tabs but the name is `nonActives`. 
2. The `if` condition guarantees that `self.$tabHeader` is not `'.' + activeTabClass`, so I don't think it can be in the set of elements in the `$tabContainer.find('.' + activeTabClass)`? Then, I think you can remove the `not`.